python/rikai/pytorch/models/resnet.py

```diff
@@ -16,27 +16,41 @@
 ResNet: Deep Residual Learning for Image Recognition
 https://arxiv.org/pdf/1512.03385.pdf
 """
-
 import torchvision
 
-from rikai.pytorch.models.torch import ClassificationModelType
+from rikai.pytorch.models.torch import (
+    classification_id_to_label_fn,
+    ClassificationModelType,
+)
 
 resnet18 = ClassificationModelType(
-    name="resnet18", pretrained_fn=torchvision.models.resnet18
+    name="resnet18",
+    pretrained_fn=torchvision.models.resnet18,
+    id_to_label_fn=classification_id_to_label_fn,
```

Comment:
can we provide a default value for `id_to_label_fn`?
Seems all classification models share the same one, similar situation applies to `object detections` as well.

Reply:
yeah, this PR have default ones setup for classification, object detection, and key point detection

Reply:
can we make that func to be a default value in `ClassificationModelType`'s constructor, so that we do need to specify it here in the most of the case.

The reason that we need `pretrained_fn` there was that each model have different method to load pretrained models.

Reply:
fixed